### PR TITLE
Add Czech translation

### PR DESCRIPTION
**Pull request by Grreby**
Only the index file is translated, I will translate the rest of the site in a few days. I read that it shouldn't break anything (hopefully). 

Signed-off-by: Jan Podhorsky <grreby@proton.me>

### CODE REVIEW

**Comment by biodranik** [comment]
It won't break anything after updating and merging #62 
Thank you for your contribution!

**Comment by Grreby** [comment]
Hi, is it possible to merge this PR? I have noticed some typos and such in my translation, so i improved it a bit. Should i upload the updated files here? Also, some things changed in the main repo, so there are merge errors. What should I do? Thanks!

File: content/news/index.cs.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
```
Comment (biodranik) [inline content/news/index.cs.md:1]: Please rename this file to _index.cs.md

It's easy to test it locally by running `zola serve`

**Comment by biodranik** [comment]
Did you fix all typos?

**Comment by Grreby** [comment]
No, not yet. I'll create the commits now.

**Comment by Grreby** [comment]
Ok, I added the improvements.

**Review by biodranik — APPROVED** [review]
Thank you very much!

**Pull request merged**